joomla/joomla-cms
# Convert mod_articles_popular to new structure

### Summary of Changes
This PR converts mod_articles_popular to the new structure

### Testing Instructions
- Use Joomla 4.3
- Create an instance of the module Articles - Most Read. Note how it works.
- Apply patch
- Delete administrator/cache/autoload_psr4.php
- Refresh the page, make sure the module still displays the same as before.
